src/components/App.js

```diff
@@ -13,16 +13,32 @@ const App = () => {
   ]
 
   const [todos, setTodos] = useState(todosList)
+  const [input, setInput] = useState('')
+
+  const submitItem = (event) => {
+    const condition = (event.key === 'Enter' || event.type === 'click') && input.length > 0
```

Comment:
`condition` es un nombre muy genérico de variable. Siempre es mejor poner nombres bien explícitos para no tener que estar leyendo el código y ver de qué trata la condición.

```suggestion
    const canAddTodo = event.key === 'Enter' && input.length > 0
```
<br>

Por otro lado, tener un event handler que se aplique a dos eventos de diferentes tipos y tenga que chequear cosas de ambos eventos es un poco confuso. Si lo seguimos extrapolando, en la medida que agreguemos más eventos tendríamos que agregar más condiciones, haciendo que el código se vuelva más difícil de leer. Lo idea sería tener dos funciones que chequeen sus propias condiciones (como `handleKeyPress`, `handleClick`) y llamen a una misma función `addTodo`.